.circleci/config.yml

```diff
@@ -86,7 +86,7 @@ jobs:
           command: sudo yarn workspace integration-tests playwright install-deps
       - run:
           name: Run Playwright tests
-          command: yarn workspace integration-tests test --workers 2 --reporter list,junit
```

Comment:
Playwright's default config runs with a single worker in CI environments. Since our config also has the CI check, I wanted to remove this override. I think running these workers in parallel is causing the browsers to crash and tests to fail on this branch.

![Screen Shot 2022-03-08 at 16 48 46](https://user-images.githubusercontent.com/630449/157345186-cf4ada16-3779-4e48-bb35-e6f44984647a.png)


If this doesn't work, we might need to increase the executor size from medium+.